xhydro/optimal_interpolation/ECF_climate_correction.py

```diff
@@ -14,6 +14,10 @@ def correction(
     centroid_lon_obs: np.ndarray,
     centroid_lat_obs: np.ndarray,
     variogram_bins: int = 10,
+    form: int = 3,
+    hmax_divider: float = 2.0,
+    p1_bnds: list = [0.95, 1],
+    hmax_mult_range_bnds: list = [0.05, 3],
```

Comment:
Règle générale, on évite d'avoir des défauts pour des listes et des dictionnaires dans Python.
```suggestion
    p1_bnds: list = None,
    hmax_mult_range_bnds: list = None,
```

Reply:
PTI! https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Encore les maudits mutable!!!